packages/hash/api/src/model/knowledge/entity.model.ts

```diff
@@ -297,6 +297,24 @@ export default class {
     return await EntityModel.fromPersistedEntity(graphApi, persistedEntity);
   }
 
+  /**
+   * Get a specific version of an entity.
+   *
+   * @param params.entityId - the id of the entity
+   * @param params.version - the version of the entity
+   */
+  static async getVersion(
+    _graphApi: GraphApi,
+    _params: {
+      entityId: string;
+      entityVersion: string;
+    },
+  ): Promise<EntityModel> {
+    throw new Error(
+      "Graph API does not yet support getting a specific version of an entity",
```

Comment:
Perhaps we should consider using unix UTC epochs instead to only allocate a number instead of a string, but that'd be a low-level optimization

Reply:
Interesting, I'll give that a try - although I don't think getting a non-latest page is a priority for a while

Reply:
Yeah, we should be a little careful with this now, as getting a non-latest page means that getblocks should be aware of the version id. There might be other queries that need to be aware of it as well. Perhaps we should leave it unimplemented, but change the reasoning

Reply:
Yeah, have rephrased in https://github.com/hashintel/hash/pull/1133/commits/ba7ede4f1d41d31be6d2c4f887a08370cec1975d.

Only reason I bothered with this btw is so that `knowledgePage`'s arguments are closer to the ones of the existing `page` GQL query (which let's you get a page at a specific version)